handsontable/src/pluginHooks.js

```diff
@@ -49,6 +68,34 @@ import { fastCall } from './helpers/function';
  *   // function body - will only run in #example2
  * });
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef1 = useRef();
+ * const hotRef2 = useRef();
```

Comment:
1 - The `hot2.addHook` section is missing - the callback is not assigned using API nor as a prop
2 - I'm not sure if assigning refs to the HotTable components without the need to reference them is a good idea
3 - I'm having trouble finding where is this JSDoc available in the docs, could you point me in the right direction?

Reply:
Ad. 3

`docs/api/hooks/`

Reply:
Ad 1.

Thanks. Fixed within: https://github.com/handsontable/handsontable/pull/9791/commits/12fd2247cd60b09fd16030f010286a80376f4cf5

Ad 2. WDYT about leaving it for showing that hook is added just to one from two instances?

Reply:
@wszymanski 
Ad 1. I _think_ in one of the meetings we discussed using the hook declarations as props, rather than `addHook` method calls, but both of them work fine, so I think that's ok 👍 

Ad 2. Now when we use the ref for `hot2` to use `addHook` it makes more sense 👍 

Ad 3. I'm trying to find a portion of the code, say, "Fired twice" at https://handsontable.com/docs/api/hooks/ and can't find anything. What am I missing?

Reply:
Ad 3. Ah, yeah. I can see that it's not rendered on current docs and on this branch.